<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>record</class>
 <widget class="QWidget" name="record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>710</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_Filename">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FileSize">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ower">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string>Ownership Claim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Time">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string>Timestamp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PublicInfo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string>More Information:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Hash">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string>Fingerprint:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileSize">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditInfo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>611</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(204, 204, 204);
background-color: rgb(244, 244, 244);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditHash">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>212</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(204, 204, 204);
background-color: rgb(244, 244, 244);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHistory">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 145, 205);
</string>
    </property>
    <property name="text">
     <string>Registration Record</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBack">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOwer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>372</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(204, 204, 204);
background-color: rgb(244, 244, 244);</string>
    </property>
   </widget>
   <zorder>label_Filename</zorder>
   <zorder>label_FileSize</zorder>
   <zorder>label_Ower</zorder>
   <zorder>label_Time</zorder>
   <zorder>label_PublicInfo</zorder>
   <zorder>labelFileName</zorder>
   <zorder>labelFileSize</zorder>
   <zorder>labelTime</zorder>
   <zorder>pushButtonHistory</zorder>
   <zorder>label</zorder>
   <zorder>pushButtonBack</zorder>
   <zorder>label_Hash</zorder>
   <zorder>textEditHash</zorder>
   <zorder>textEditOwer</zorder>
   <zorder>textEditInfo</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
